db/migrate/20160429133701_create_games.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+      t.string :status
+      t.integer :moves
+      t.text :board
```

Comment:
not sure what you mean? by http://www.jsoneditoronline.org/?id=caef066ba81c4f918f20d758bcd7bd11

that's still a `json` object, we ought to leverage `postgres`'s features